libr/anal/type_dwarf.c

```diff
@@ -46,8 +46,9 @@ static void enum_type_fini(void *e, void *user) {
  * @return st32 Index, -1 if nothing found
  */
 static st32 find_attr_idx(const RBinDwarfDie *die, st32 attr_name) {
+	st32 i;
```

Comment:
Why signed? Use size_t

Reply:
Oh i see.. it returns -1 on error. Better use int. So it will fit arch reg size instead. Anyway the pr lgtm as it fixes the build